src/imitation/algorithms/preference_comparisons.py

```diff
@@ -812,7 +816,14 @@ def __init__(
 
         self.dataset = PreferenceDataset()
 
-    def train(self, total_timesteps: int, total_comparisons: int) -> Mapping[str, Any]:
+        self.vec_normalize = vec_normalize
```

Comment:
This attribute doesn't seem to actually be used anywhere?

Reply:
You are right about it!